### PR TITLE
Added Option to Place Limit Line Label Beneath Line

### PR DESCRIPTION
**Pull request by dtrauger**
Added option to place limit line label beneath limit line adjusting the line's height accordingly.

### CODE REVIEW

**Comment by thierryH91200** [comment]
@dtrauger 

You are sure that update project has a relationship to the problem

refer to #1538 

I quote petester42 
"Yes you should. It's is one framework for all platforms. The basic setup is macOS but it has been setup to work on all platforms. Having 1 framework makes the development more manageable."

It's dangerous your PR

**Comment by petester42** [comment]
As mentioned the project file should be macOS not iphoneos. Please revert the change

File: Source/Charts/Renderers/BarChartRenderer.swift [hunk 1]
```diff
@@ -68,6 +68,24 @@ open class BarChartRenderer: BarLineScatterCandleBubbleRenderer
         }
     }
     
+    open func calcWidth(span: CGFloat) -> CGFloat
+    {
+        var width = CGFloat(60.0)
```
Comment (jjatie) [inline Source/Charts/Renderers/BarChartRenderer.swift]: Where do these numbers come from?

**Comment by jjatie** [comment]
Feel free to reopen if you'd make the requested changes.

**Pull request closed without merging**